### PR TITLE
Update foodon.md

### PR DESCRIPTION
**Pull request by ddooley**
Updating usages

### CODE REVIEW

**Review by matentzn — COMMENTED** [review]
Thank you @ddooley for updating the impressive usages for FOODON!

Unfortunately, so we dont open the floodgates to the wild west, we require verifiable usages. For example: 

https://github.com/OBOFoundry/OBOFoundry.github.io/blob/0881025c7a5fba763b935c0eb656e2130073a38f/ontology/mondo.md?plain=1#L67

A link to a FOODON term on a site is a good proof.

We have had huge debates whether scientific papers count as proof and I forgot were we stand on this, but I guess we said ok.

**Comment by ddooley** [comment]
I was trying to fulfill usages references for FoodOn, but I can't really see what's wrong in yaml file from error report of first step. Also I don't know how to edit what I submitted there from the OBO Foundry "Edit" yaml option which leads to: https://github.com/OBOFoundry/OBOFoundry.github.io/edit/master/ontology/foodon.md

**Comment by matentzn** [comment]
You can 

1. https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2609/files click here
2. Hit the edit button:

image

To edit the yaml file in this PR.

Dont worry about QC, for now, I can help you fix it. You have to run "tox -e lint" on the command line, but @jsstevenson will write an action at some point to do this automatically. In the meantime, I can do it for you.

**Comment by ddooley** [comment]
Ok thanks for your help.  I've updated the pull request to include links to specific entries showing FoodOn IDs.

File: ontology/foodon.md [hunk 1]
```diff
@@ -37,6 +37,27 @@ repository: https://github.com/FoodOntology/foodon
 tags:
 - food
 tracker: https://github.com/FoodOntology/foodon/issues/
+usages:
```
Comment (deepakunni3) [inline ontology/foodon.md:40]: @ddooley I took the liberty of elaborating the description for most of the usages of FOODON.
Please feel free to reword as appropriate.

File: ontology/foodon.md [hunk 2]
```diff
@@ -37,6 +37,27 @@ repository: https://github.com/FoodOntology/foodon
...
+    url: https://fdc.nal.usda.gov/fdc-app.html#/food-details/1750340/attributes
+  user: https://fdc.nal.usda.gov/
+- description: FDA GenomeTrackr surveillance program for reporting foodborne pathogen biosamples.
+  examples:
+  - description: An entry from NCBI Biosample that describes a *Samonella enterica* sample extracted from Chicken and annotated with the term FOODON:03411457.
+    url: https://www.ncbi.nlm.nih.gov/biosample/SAMN03455272
+  user: https://www.fda.gov/food/whole-genome-sequencing-wgs-program/genometrakr-network
+- description: Periodic Table Of Food Initiative database of food biosample nutrition analytics.
+  examples:
+  - description: <TODO>
```
Comment (deepakunni3) [inline ontology/foodon.md:53]: @ddooley Unfortunately I was not able to explore this particular example since it requires a login.

Comment (ddooley) [reply ontology/foodon.md:53]: Ok, lets drop PTFI then.  Its free access but login required.  Maybe later they'll make it without login.

**Review by deepakunni3 — APPROVED** [review]

**Comment by deepakunni3** [comment]
@ddooley Thank you! Looks good to me :)

**Pull request merged**